2019/2019-10-13-brainstorming.md

```diff
@@ -0,0 +1,122 @@
+# SR KTC Brainstorming Day 2019-10-13
+
+Date: Sunday 13th October 2019, 10:00
+
+Present: Kajetan Champlewski, Kier Davis, Rob Gilton, Dan Trickey, Tom Wheal
+
+## Minutes
+
+* Values of SR
+  * Accessible
+  * Autonomous
+  * Open by default
+  * Encourage creativity
+  * Reflect reality
+* What does it mean to have a vision?
+  * Everyone acts in a way to further us towards a common goal
+  * A vision should not be expected to be totally achieved - it's an ideal.
+  * We should be regularly adjusting the vision to ensure we're constantly moving.
+  * Ideas:
+    * {Maximising,fulfilling,empowering} competitor {learning,development,experience,potential}
+    * Life, liberty and the pursuit of robots
+    * Help students {learn,discover their ability} to {make,engineer} amazing robots
+    * Industry class, student built
+      * Students are exposed to real world technologies with a carefully chosen level of abstraction
+    * Bringing creativity and engineering to the surface
+    * Open-ended, no correct answer
+    * Real world robotics
+    * Enabling, not teaching
+  * Best one for now: "Empowering teams to discover their ability to engineer amazing robots"
+* We have a number of stakeholders in our products/services:
+  * Competitors: actually using the kit first-hand
+  * Competition team: functionally a middleman, but they give the kit an application/purpose by building a competition around it
+  * Volunteers in general: interacting with the kit comes naturally as part of supporting teams
+* What are the attributes of a kit service that implements this vision?
+  * Easily enabling a minimal viable robot / minimal awesome robot
+  * Kit as a catalyst for students' imagination
+  * Expectations matched to available resources
+  * Kit works out of the box with a team of students (in regard to resources available at school)
+  * Open-ended
+  * Experience themselves
+  * No prescribed process
+  * Should be amazing for students
+  * Should be amazing for spectators
+* What is a subteam?
+  * Group of people collaborating on a piece of work.
+  * No more than about 6 people.
+  * Subteams should have freedom to organise themselves how they see fit.
+  * Committee should oversee operation of the subteam to ensure they are making reasonable progress.
+  * Should keep in touch with the committee on a regular basis, so that they can express their progress and any issues they're having, and we can offer ways to help.
+  * Idea: conferences - each subteam gets a chance to present their work to the rest of the kit team.
+    * Aim to build a sense of community among the whole kit team
+  * Idea: service manuals produced by development teams for use by asset management teams.
+* Ideas for subteams:
+  * Infrastructure team
+    * Provide and maintain managed hosting services to the rest of the organisation
```

Comment:
The infrastructure doesn't fall under the remit of the kit team, or either team for that matter, for it is not kit.

That being said, The applications *on* the infrastructure may fall under the remit of either team, most of which is likely under the domain of the kit team.

Reply:
Jake, please wait until we announce this before jumping to any conclusions.